build/azure-pipelines/build-common.yml

```diff
@@ -48,8 +48,8 @@ steps:
   displayName: 'Go: build sqlcmd'
   inputs:
     command: 'build'
-    arguments: '-o $(Build.BinariesDirectory) -ldflags="-X main.version=${{ parameters.VersionTag }}"'
-    workingDirectory: '$(Build.SourcesDirectory)/cmd/sqlcmd'
+    arguments: '-o sqlcmd.exe $(Build.BinariesDirectory) -ldflags="-X main.version=${{ parameters.VersionTag }}"'
```

Comment:
>sqlcmd.exe  [](http://example.com/codeflow?start=19&length=11)

the binary name is dependent on GOOS and probably needs to get added to the variable matrix

Reply:
and we want the binary to end up in $(Build.BinariesDirectory)